python/lsst/eotest/image_utils.py

```diff
@@ -223,10 +328,13 @@ def bias_image(im, overscan, dxmin=5, dxmax=2, statistic=np.mean, bias_method='r
     Returns:
         An image with size equal to the input image containing the offset level.
     """
-    if bias_method not in ['mean', 'row', 'func', 'spline']:
-        raise RuntimeError('Bias method must be either "mean", "row", "func" or "spline".')
-    method = {'mean' : bias, 'row' : bias_row, 'func' : bias_func, 'spline' : bias_spline}
-    my_bias = method[bias_method](im, overscan, dxmin=dxmin, dxmax=dxmax, **kwargs)
+    if bias_method.lower() not in ['mean', 'row', 'func', 'spline', 'none']:
```

Comment:
I would generally prefer `None` instead of `'none'` (and I wouldn't do the `.lower()` conversion).  I think it would be confusing to use the string `'none'` (or `'None'`) when `None` is a ubiquitous default value for keyword options.

Reply:
In principle I agree with using None instead of 'none',   However, I'm just grabbing the value from either a command line option or a yaml file and using the value to construct output file names, so overall it is simpler to keep it as a string everywhere, so it would save a fair bit or work to leave that as is.    The 'lower()' is to support some very old superbias files, (i.e., the file names have _Spline_ in them, and it is easier if that matches the option here.)   I can check if those files are still around or just change the names if you feel strongly about that.